data-access-analysis-tools/portal-intro.rst

```diff
@@ -394,3 +394,79 @@ The color-composite background shows the relevant DC2 simulated sky region.
 
 **Learn More.**
 See also :ref:`DP0-2-Tutorials-Portal` for a tutorial using additional image types and more of the Portal's image-related functionality.
+
+.. _Portal-Intro-URL-APIs:
+
+Public URL API for the Portal
+=============================
+
+The public URL API provides direct access to specific resources via a publicly accessible URL; however, an RSP login is still required.
+
+One application is to construct URLs for your ADQL queries, enabling you to share these links with collaborators for efficient
+communication or save them for future reuse, avoiding the need to repopulate the ADQL edit box each time. Here are some examples.
+
+**1. Exploring the Object table**
+
+To learn about the contents of a tableto, retrieve the first 100 rows of the Object table using the SELECT TOP statement.
+
+.. code-block:: SQL
+
+    SELECT TOP 100 * FROM dp02_dc2_catalogs.Object
+
+Corresponding URL API: https://data.lsst.cloud/portal/app/?api=tap&service=https://data.lsst.cloud/api/tap&adql=SELECT%20TOP%20100%20*%20FROM%20dp02_dc2_catalogs.Object&execute=true
+
+**2. Populate the ``UI-assisted'' page for a cone search**
+
+The URL below populates the "UI-assisted" page to perform a cone search on the Object catalog using a radius of 0.05 degrees 
+centered at (RA, Dec) = (62, -37).
+
+URL API: https://data.lsst.cloud/portal/app/?api=tap&service=https://data.lsst.cloud/api/tap&schema=dp02_dc2_catalogs&table=dp02_dc2_catalogs.Object&ra=62.0&dec=-37.0&sr=0.05d
+
+.. figure:: /_static/portal_intro_DP02l.png
+    :name: portal_populatedUIpage
+    :alt: A screenshot of the populated UI-assisted page for a cone search of the Object catalog, with a 0.05 degree radius around (ra,dec)=(62,-37)
+
+**3. Execute the cone search above**
+
+You can also perform this search by editing the ADQL query box and clicking the search button. Below is the ADQL query for
+the cone search, along with the URL API that executes the search directly.
+
+.. code-block:: SQL
+
+   SELECT coord_dec, coord_ra
+   FROM dp02_dc2_catalogs.Object
+   WHERE CONTAINS(POINT('ICRS', coord_ra, coord_dec),
+   CIRCLE('ICRS', 62, -37, 0.05)) = 1
+
+Corresponding URL API: https://data.lsst.cloud/portal/app/?api=tap&service=https://data.lsst.cloud/api/tap&adql=SELECT%20coord_dec,coord_ra%20FROM%20dp02_dc2_catalogs.Object%20WHERE%20CONTAINS(POINT('ICRS',coord_ra,coord_dec),CIRCLE('ICRS',62,-37,0.05))%3D1&execute=true
```

Comment:
Please put this link (and the ones below) under text, as currently it is just displaying like this and it does not look good: 
<img width="1105" alt="image" src="https://github.com/user-attachments/assets/6dfd2941-72a3-48e5-a0b9-95752a8a0a2a">

Reply:
@plazas I agree they look ugly this way, but the purpose of explicitly displaying the URLs is to offer guidance on how to construct them. Maybe in the intro text of this section, I should explicitly state that. What do you think?

Reply:
I modified the intro text and put the URLs in a better format using the .. code-block:: text.

Reply:
Thanks, yeah, I was going to suggest something like that!